frontend/src/platform/types/user.ts

```
@@ -0,0 +1,76 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+/* This file was automatically generated from pydantic models by running pydantic2ts.
+/* Do not modify it by hand - just update the pydantic models and then re-run the script
+*/
+
+export interface SQLModel {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)